app/src/main/java/net/gsantner/opoc/util/StringUtils.java

```diff
@@ -345,4 +347,35 @@ public static void selectLines(final EditText edit, final List<Integer> position
             edit.setSelection(selStart, selEnd);
         }
     }
+
+    // Search for matching pairs of backticks
+    // interpolate contents of backtick pair as SimpleDateFormat
+    public static String interpolateEscapedDateTime(final String snip) {
```

Comment:
I suggest this: `date: "<<parsed format>>"`

(Somewhat in sense of <https://learn.cloudcannon.com/jekyll/date-formatting#date>)

Reply:
Hmmm, I was going with something inspired by textmate snippets

Reply:
Using backticks is easier to parse and escape IMO. We can use a type prefix: `\`date: HHmm\`` to differentiate interpolation types if we want multiple types